src/util.c

```diff
@@ -611,16 +607,16 @@ int write_result_into_json_file(struct ntttcp_test_endpoint *tep)
 	fprintf(json_file, "            \"metric\" : \"cycles/byte\",\n");
 	fprintf(json_file, "            \"text\" : \"%.3f\"\n", tepr->cycles_per_byte);
 	fprintf(json_file, "        },\n");
-	fprintf(json_file, "        \"packetsSent\" : \"%"PRIu64"\",\n", tepr->packets_sent);
-	fprintf(json_file, "        \"packetsReceived\" : \"%"PRIu64"\",\n", tepr->packets_received);
-	fprintf(json_file, "        \"packets_retransmitted\" : \"%"PRIu64"\",\n", tepr->packets_retransmitted);
+	fprintf(json_file, "        \"packetsSent\" : \"%" PRIu64 "\",\n", tepr->packets_sent);
```

Comment:
Please double check the " " space in front of the PRIu64.

Reply:
done